tests/backstop_data/engine_scripts/puppet/mosaico/mailing/editor-test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await require('./step-2.js') (engine, scenario, viewport);
+  await engine.waitFor('a[title="Edit"]');
+  // wait for Angular to bind function
+  await engine.waitFor(200);
```

Comment:
can't be like that, with an arbitrary amount of time. Isn't there anything, UI-wise, that tells you if angular is ready?